.gitignore

```diff
@@ -16,3 +16,7 @@ xcuserdata
 DerivedData
 *.xcuserstate
 node_modules/
+
+### macOS Finder ###
```

Comment:
this is should be in local gitignore file

Reply:
left .gitignore unchanged and force-pushed